my_game.py

```diff
@@ -80,8 +83,11 @@ def __init__(self):
             scale=SPRITE_SCALING
         )
 
+        self.angle = random.randint(-90, 180)
         self.center_x = random.randint(0, SCREEN_WIDTH)
         self.center_y = random.randint(0, SCREEN_HEIGHT)
+        self.change_x = math.sin(math.radians(self.angle))
+        self.change_y = math.cos(math.radians(self.angle))
```

Comment:
`math.sin/math.cos` work with radians. You are giving it degrees. Looks like you should be able to use `self.radians` -> https://api.arcade.academy/en/latest/api/sprites.html#arcade.Sprite.radians. Make sure it works correctly by trying out angles like 90, 180 45

Also, please use aour new variable `ASTEROIDS_SPEED`